legal-api/src/legal_api/resources/v2/business/business.py

```diff
@@ -49,7 +50,11 @@ def get_businesses(identifier: str):
     if not business:
         return jsonify({'message': f'{identifier} not found'}), HTTPStatus.NOT_FOUND
 
-    return jsonify(business=business.json())
+    business_json = business.json()
+    recent_filing_json = CoreFiling.get_most_recent_filing_json(business.id, None, jwt)
+    if recent_filing_json:
+        business_json['submitter'] = recent_filing_json['filing']['header']['submitter']
```

Comment:
@thorwolpert I don't see `lastModified` in business is updating from entity-filer. Should I replace that with filing date here?

Reply:
yes please